src/aat/resources/application.yml

```diff
@@ -0,0 +1,36 @@
+---
+spring:
+  main:
+    allow-bean-definition-overriding: true
+
+test:
+  url: ${TEST_URL:http://localhost:8080}
+
+idam:
+  api:
+    url: ${IDAM_API_BASE_URI:http://localhost:4501}
+  client:
+    id: ${OAUTH_CLIENT:webshow}
+    secret: ${FUNCTIONAL_TEST_CLIENT_OAUTH_SECRET:AAAAAAAAAAAAAAAA}
+    redirect_uri: ${IDAM_WEBSHOW_WHITELIST:http://localhost:8080/oauth2redirect}
+
+s2s:
+  api:
+    url: ${S2S_BASE_URI:http://localhost:4502}
+    secret: ${FUNCTIONAL_TEST_CLIENT_S2S_TOKEN:AAAAAAAAAAAAAAAA}
+    serviceName: ${S2S_SERVICE_NAME:em_gw}
+
+ccd-def:
+  api:
+    url: http://localhost:4451
+
+document_management:
+  url: docker ps
```

Comment:
what does this do

Reply:
it's a mistake. it should be URL to DM Store